core/worker.cc

```diff
@@ -281,8 +288,14 @@ void *run_worker(void *_arg) {
   return ctx.Run(_arg);
 }
 
-void launch_worker(int wid, int core) {
-  struct thread_arg arg = {.wid = wid, .core = core};
+void launch_worker(int wid, int core, [[maybe_unused]] const std::string &scheduler) {
+  SchedulerType scheduler_type = SchedulerType::FAST;
```

Comment:
Can we check `scheduler` here (even in a hardcoded way), rather than deferring error check until `Worker::Run()`?
```
if (scheduler == "fast") {
  arg.scheduler = new FastScheduler<Task>();
} else {
  CHECK(0);
}
```

My concerns are: 1. a new type `SchedulerType` is introduced without much necessity, 2. the argument "scheduler" here is not being used, 3. `Worker::Run()` is too late to handle error gracefully (notifying the client that invalid scheduler is specified).